site/docs/hosting/comparison.md

```diff
@@ -36,10 +36,10 @@ The main thing to know about them is that on serverless infrastructures you are
 | Fly                    | Free       | $1.94/mo subscription for shared-cpu-1x and 256 MB RAM, $0.02/GB network                                   | [3 shared-cpu-1x 256mb VMs, 160GB/mo, 3GB storage](https://fly.io/docs/about/pricing/)  | ✅      | ✅                           | ❓  |                                                                                                  |
 | DigitalOcean Functions | Free       | $1.85/100K GB-s                                                                                            | [90K GB-s/mo](https://docs.digitalocean.com/products/functions/details/pricing/)        | ✅      | ❌                           | ❓  |                                                                                                  |
 | Cloudflare Workers     | Free       | $5/10M req                                                                                                 | [100K req/day, 10 ms CPU-time limit](https://workers.cloudflare.com/)                   | ❌      | [✅](https://denoflare.dev/) | ✅  |                                                                                                  |
-| Vercel                 | Free       | $20/mo subscription                                                                                        | [Unlimited invocations, 100 GB-h, 10 s time limit](https://vercel.com/pricing)          | ❓      | ❓                           | ❓  | Not intended for non-websites?                                                                   |
+| Vercel                 | Free       | $20/mo subscription                                                                                        | [Unlimited invocations, 100 GB-h, 10 s time limit](https://vercel.com/pricing)          | [✅](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js)      | [✅](https://github.com/vercel-community/deno)                           | [✅](https://vercel.com/docs/frameworks)  | Only webhooks supported                                                                   |
```

Comment:
I think all platforms listed here only support webhooks. There's no need to mention it here.

Reply:
Updated